Lib9c/Action/Stake.cs

```diff
@@ -104,13 +104,13 @@ public override IAccountStateDelta Execute(IActionContext context)
 
             var latestStakeContract = new Contract(
                 stakeRegularFixedRewardSheetTableName:
-                    stakePolicySheet["StakeRegularFixedRewardSheet"].Value,
+                stakePolicySheet["StakeRegularFixedRewardSheet"].Value,
                 stakeRegularRewardSheetTableName:
-                    stakePolicySheet["StakeRegularRewardSheet"].Value,
+                stakePolicySheet["StakeRegularRewardSheet"].Value,
                 rewardInterval:
-                    long.Parse(stakePolicySheet["RewardInterval"].Value, CultureInfo.InvariantCulture),
+                long.Parse(stakePolicySheet["RewardInterval"].Value, CultureInfo.InvariantCulture),
                 lockupInterval:
-                    long.Parse(stakePolicySheet["LockupInterval"].Value, CultureInfo.InvariantCulture)
+                long.Parse(stakePolicySheet["LockupInterval"].Value, CultureInfo.InvariantCulture)
```

Comment:
Is it intended? 👀

Reply:
Oh, I didn't intended. It was automatically formatted. And.. it'll be changed.😉